drivers/include/adxl345.h

```diff
@@ -120,29 +130,31 @@ typedef struct {
     uint8_t time_inact;   /**< Inactivity time */
     uint8_t thres_ff;     /**< Free-fall threshold */
     uint8_t time_ff;      /**< Time threshold */
-    uint8_t act_inact;    /**< Axis enable control for activity and inactivity detection */
+    uint8_t act_inact;    /**< Enable ctrl for activity/inactivity detection */
     uint8_t tap_axes;     /**< Axis control for single tap/double tap */
 } adxl345_interrupt_t;
 
 /**
  * @brief Configuration struct for the ADXL345 sensor
  */
 typedef struct {
-    i2c_t i2c;                     /**< I2C device which is used */
-    uint8_t addr;                  /**< I2C address */
-    adxl345_interrupt_t interrupt; /**< Interrupts configuration  */
-    uint8_t offset[3];             /**< offset axis */
-    uint8_t range;                 /**< Sensitivity configuration */
-    uint8_t rate;                  /**< Configured sample rate for accel  */
-    uint8_t full_res;              /**< Resolution bit */
-    uint8_t scale_factor;          /**< Scale factor for converting value to mg */
```

Comment:
I don't know the device in detail, but it seems to me, like some of these fields are static configuration values, while others can be deducted from them, right?
`interrupt` -> dynamic param set at runtime -> belongs into `adx1345_t`
`offset` -> static params, part of `adx1345_params_t`
`range` -> static param, part of `adx1345_params_t`
`rate` -> static param, part of `adx1345_params_t`
`full_res` -> deducted from `range`?! -> goes into device descriptor
`scale_factor` -> deducted from `range` -> goes into device descriptor